src/inverseRobot.js

```diff
@@ -24,7 +24,16 @@
  * @return {object}
  */
 function inverseRobot(robot) {
-  // write code here
+  const reversedRobot = {};
+
+  for (const key in robot) {
+    if (reversedRobot[robot[key]] !== undefined) {
```

Comment:
```suggestion
    if (reversedRobot[robot[key]]) {
```
Якщо в новому роботі ти знайдеш щось по ключу то отримаєш true і зайдеш в if але якщо ключа ще немає то отримає undefined який перетвориться в false і пропустиш цей іф. тому порівняння з undefined тут зайве